tests/tests_integration/test_api/test_simulators/test_integrations.py

```diff
@@ -30,7 +29,7 @@ def test_list_integrations(self, cognite_client: CogniteClient) -> None:
         assert len(integrations) > 0
 
     def test_filter_integrations(self, cognite_client: CogniteClient, seed_resource_names) -> None:
-        for integration in cognite_client.simulators.integrations(filter=SimulatorIntegrationFilter(active=True)):
+        for integration in cognite_client.simulators.integrations.list(active=True):
```

Comment:
```suggestion
        for integration in cognite_client.simulators.integrations(active=True):
```
we want to test __call__ here